persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/configuration/TableManipulationConfigurationBuilder.java

```diff
@@ -37,9 +37,9 @@
    }
 
    /**
-    * Repetitive DB operations this are batched according to this parameter. This is an optional parameter, and if it
-    * is not specified it will be defaulted to {@link TableManager#DEFAULT_BATCH_SIZE}.
+    * Deprecated. Please use {@link org.infinispan.configuration.cache.AbstractStoreConfigurationBuilder#maxBatchSize(int)} instead.
     */
+   @Deprecated
    public S batchSize(int batchSize) {
       attributes.attribute(BATCH_SIZE).set(batchSize);
```

Comment:
I meant that the config would directly update `maxBatchSize` and use that when it creates the `TableManager`. There is no reason to store the same config option in 2 places.